tests/integration/test_explain.py

```diff
@@ -0,0 +1,30 @@
+import dask.dataframe as dd
+import pandas as pd
+import pytest
+
+
+@pytest.mark.parametrize("gpu", [False, pytest.param(True, marks=pytest.mark.gpu)])
+def test_sql_query_explain(c, gpu):
+    df = dd.from_pandas(pd.DataFrame({"a": [1, 2, 3]}), npartitions=1)
+    c.create_table("df", df, gpu=gpu)
+
+    sql_string = c.sql("EXPLAIN SELECT * FROM df")
+
+    assert sql_string.startswith("Projection: #df.a\n")
+
+    # TODO: Need to add statistics to Rust optimizer before this can be uncommented.
```

Comment:
Noted and understood

Reply:
For context: I just copied over some unit tests from `c.explain` and this comment was originally from there. The reason I left it in here as well is because we might want to explore this for both `c.explain` and `c.sql("EXPLAIN ...")` when they get added.